### PR TITLE
Made footer

### PR DESCRIPTION
**Pull request by Akshay-Jak**
Signed-off-by: Akshay-Jak <akshayjani2147@gmail.com>

### Functionality:
Made the footer component added icons 

### Solution:
Made a footer component

### Risk level:
- [ ] high 
- [x] medium
- [ ] low

### How to test:
ng serve 
firebase emulators:start

Screenshot (1129)

### CODE REVIEW

**Review by arjund-06 — APPROVED** [review]
Ship it

**Comment by viv9k** [comment]
Ui is not looking good, for footer... Let's connect and discuss over it.

File: src/app/app.component.css [hunk 1]
```diff
@@ -1,6 +1,15 @@
 .app-theme {
     min-height: 100vh;
     background-color: var(--primary-bg);
+    position: relative;
+}
+
+.wrap {
+    padding-bottom: 10vh;
+}
```
Comment (viv9k) [inline src/app/app.component.css:9]: This class is not needed now.

**Review by viv9k — CHANGES_REQUESTED** [review]
Please fix the required changes

**Review by viv9k — APPROVED** [review]
Ship it

**Pull request merged**